test/page.spec.ts

```diff
@@ -37,13 +37,17 @@ describe('Page', function () {
       const { context } = getTestState();
 
       const newPage = await context.newPage();
+      const evaluatePromise = newPage
+        .evaluate(() => new Promise(() => {}))
+        .catch((error_) => (error = error_));
+
+      await newPage.evaluate(() =>
+        console.log('make sure that the previous eval got to the browser')
+      );
+
       let error = null;
-      await Promise.all([
-        newPage
-          .evaluate(() => new Promise(() => {}))
-          .catch((error_) => (error = error_)),
-        newPage.close(),
```

Comment:
I would be more in favor of getting this fixed in Firefox.

Since when is it actually failing? Is it triggered by a recent change in Puppeteer? I would be interested in a larger trace log. Maybe you can give some steps to reproduce? Thanks.

Reply:
I’m implementing Firefox support on PuppeteerSharp (https://github.com/hardkoded/puppeteer-sharp/pull/1640)

I found that this test was passing here but failing in pptr#.
While debugging this test here, you can do the same in the main branch,
I found that the test was passing, but the “protocol error” I was getting wasn't due to the “target closed” error generated “gracefully” by pptr, but in fact, due to the error I mentioned in the description.

If that can be fixed upstream, perfect :)